### PR TITLE
Checklist recipe blog post

### PR DESCRIPTION
**Pull request by peterdesmet**
Hi @stefaniebutland @sckott, here's our blog post on the Ebbe Nielsen Challenge-winning Checklist recipe. As discussed, we focus on some of the tools we discovered in developing it, so it is a bit tutorial-like in some aspects. Some notes for editorial (you might want to change):

* I used [Oxford spelling](https://en.wikipedia.org/wiki/Oxford_spelling)
* I used abbreviated `we'd`, `don't`
* ~~I started titles at `h2`, but noticed some other blog posts started at `h3`: what is the rule for the rOpenSci blog?~~ -> seeing the preview I updated this to h3
* Quoted text uses simple quotes `"`, not curly ones.

### CODE REVIEW

File: content/blog/2018-11-20-checklist-recipe.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+Anyone can [publish species information](https://www.gbif.org/publishing-data) (called "checklist data") to GBIF. When you do, GBIF will create a page for your dataset ([example](https://doi.org/10.15468/xvuzfh)), assign a DOI, and match[^1] your scientific names to its [backbone taxonomy](https://doi.org/10.15468/39omei), allowing your data to be linked, discoverable and integrated. Species pages on GBIF for example ([example](https://www.gbif.org/species/5851603)) are automatically build from the over [25.000](https://www.gbif.org/dataset/search?type=CHECKLIST) checklist datasets that have been published. All your checklist information also becomes available via the GBIF API ([example](https://api.gbif.org/v1/species/141117238)) and can be queried using the [rgbif](https://github.com/ropensci/rgbif) package.
```
Comment (sckott) [inline content/blog/2018-11-20-checklist-recipe.md]: "automatically built"

File: content/blog/2018-11-20-checklist-recipe.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+We use this information to verify if our scientific names are indeed written as scientific names and to populate the taxon rank (a mandatory Darwin Core term for checklists) using `rankMarker`. Note that the name parser does not check the existence of a scientific name against an existing registry. That is done by the [GBIF species lookup](https://www.gbif.org/tools/species-lookup) tool we discussed above, which verifies the existence of a name in the GBIF backbone taxonomy.
+
+[rgbif](https://github.com/ropensci/rgbif) provides many more functions to interact with the Global Biodiversity Information Facility and we use the package extensively for our [TrIAS project](https://github.com/trias-project).
+
+### Conclusion
+
+Our recipe grew organically from experience we gained publishing data to GBIF. We saw the GBIF Ebbe Nielsen Challenge as an opportunity to bottle and document what we had learned in an opinionated template to help others and we hope this blog post highlighted a few tips and tricks that might be useful to you as well. If want to use the recipe to publish your own checklist data, [start here](https://github.com/trias-project/checklist-recipe/wiki).
```
Comment (sckott) [inline content/blog/2018-11-20-checklist-recipe.md]: "If you want to use the recipe"

**Comment by stefaniebutland** [comment]
I'm reviewing now too - comments shortly (this is a fantastic post!!)

**Comment by peterdesmet** [comment]
@stefaniebutland Gee, thanks for this very positive review! 😊

I've made the necessary typo corrections and used the new twitter notation + our ORCID as main URL for the authors.

A tweet with the handle of each author + @LifeWatchINBO sounds great. If you can fit it in, [@trias_project](https://twitter.com/trias_project) would also be great to include, since the recipe was developed in the framework of that project.

Looking forward to see this published! ✨

**Comment by peterdesmet** [comment]
Has this been tweeted yet? 🤔 Would like to retweet. 😊

**Comment by stefaniebutland** [comment]
Not yet! I'm drafting a short thread. I'll tag you in the tweet as discussed. Thanks for your patience

**Comment by stefaniebutland** [comment]
Done. Thank you again for writing this post for the rOpenSci blog and for your donation to NumFOCUS! I hope this gets even more eyes on your work

**Pull request merged**